pages/entropy/generate-random-numbers/evm.mdx

```diff
@@ -200,3 +161,20 @@ Check the [Current Fees](../current-fees) to find the current fee for each provi
 ### Best Practices
 
 Check out the [Best Practices](../best-practices) guide for tips to limit gas usage, or generate multiple random numbers in a single transaction.
+
+<Callout type="info">
```

Comment:
I think the callout here needs some wordsmithing because you don't always need to select a provider. Probably something like "Some methods on Entropy require selecting..."